scripts/format.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+  # format source files
+  yapf -i -r --style '{based_on_style: pep8, column_limit: 120}' securedrop bin
```

Comment:
I think it would be better to put the style spec in `setup.cfg`. This file can be used for configuring other stuff as well (such as packaging information), so it would make sense to have a centralized source this kind of info. See [here](https://github.com/google/yapf#formatting-style).

Reply:
Put the styling in a setup.cfg file. The setup.cfg file lives in the same directory as format.sh.